pkg/version/parse_test.go

```diff
@@ -0,0 +1,38 @@
+package version
+
+import (
+	"github.com/blang/semver"
```

Comment:
i think we are using another semver package already, can we use the same one here, rather than introduce another dep?

Reply:
removed